addons/arsenal/functions/fnc_fillSort.sqf

```diff
@@ -89,11 +93,17 @@ private _lastSort = [GVAR(lastSortLeft), GVAR(lastSortRight)] select _rightSort;
 private _sortIndex = 0;
 
 {
-    if (_x isEqualTo []) exitWith {};
+    if (_x isEqualTo []) then {
```

Comment:
I'm unsure about this change, but to me it seems rather odd to stop at an empty array (as it was previously).

Reply:
Seems correct. continue didn't exist back then

Reply:
> Seems correct. continue didn't exist back then

I would have imagined the person who made it back then would have just put the bracket like this:
```SQF
{
    if (count _x != 0) then { // or !(_x isEqualTo [])
        _x params ["_sortName", "_displayName", "", "_condition"];
        // Insert rest here
    };
} forEach _sorts;
```

I just don't understand why it exited instead of skipping the empty element.

Reply:
The updated code doesn't seem to pose a problem, so going with it for now.